src/tests/system/tests/test_ipa_trusts.py

```diff
@@ -60,3 +62,132 @@ def test_ipa_trusts__lookup_group_without_sid(ipa: IPA, trusted: GenericADProvid
     status = ipa.sssctl.domain_status(trusted.domain, online=True)
     assert "online status: offline" not in status.stdout.lower(), "AD domain went offline!"
     assert "online status: online" in status.stdout.lower(), "AD domain was not online!"
+
+
+@pytest.mark.importance("medium")
+@pytest.mark.ticket(jira="RHEL-14752")
+@pytest.mark.topology(KnownTopology.IPATrustIPA)
+def test_ipa_trusts__ipa_master_lookup_trusted_user(ipa: IPA, trusted: IPA):
```

Comment:
Technically, this test does two things, lookup a user and group, 

```test_ipoa_trusts__ipa_master_lookup_trusted_user_and_group``` ? 

Add one more step in the docstring to look up the group.

Reply:
Renamed and updated docstring steps.